packages/openapi-fetch/test/index.test.ts

```diff
@@ -1,33 +1,137 @@
-// @ts-expect-error
-import createFetchMock from "vitest-fetch-mock";
+import {
+  http,
+  HttpResponse,
+  type JsonBodyType,
+  type StrictResponse,
+  type StrictRequest,
+  type DefaultBodyType,
+  type HttpResponseResolver,
+  type PathParams,
+  type AsyncResponseResolverReturnType,
+} from "msw";
+import { setupServer } from "msw/node";
 import createClient, {
   type Middleware,
   type MiddlewareRequest,
   type QuerySerializerOptions,
 } from "../src/index.js";
 import type { paths } from "./fixtures/api.js";
 
-const fetchMocker = createFetchMock(vi);
+const server = setupServer();
+
+/**
+ * Default baseUrl for tests
+ */
+const baseUrl = "https://api.example.com";
+
+/**
+ * Test path helper, returns a an absolute URL based on
+ * the given path and base
+ */
+function testPath(path: string, base: string = baseUrl) {
```

Comment:
This is only an internal helper, but could probably be re-named to something more descriptive.